### ISSUE TITLE
[CODE FREEZE] master branch not accepting pull requests

### ISSUE THREAD

**Issue by raisedadead** [freeCodeCamp/freeCodeCamp#38308]
Hello Everyone!

This issue is used to announce code freeze on branches as and when needed. If you find this issue open and pinned on the repo, the code freeze in effect.

<details>

<summary><strong>What is a code freeze?</strong></summary>

<br>

A code freeze is when we are not accepting any pull-requests without the approval of the @freeCodeCamp/dev-team. This is because we are in the middle of a major change on the repo and want to avoid any inconsistency in the code.

This limitation is most likely temporary and will be lifted in  a few days.

Thanks a lot for your patience and understanding. Please reach out to [contributors chat room](https://gitter.im/FreeCodeCamp/Contributors) for additional details and confirmation on the current code freeze.

</details>

**Comment by raisedadead** [comment]
Code Freeze for https://github.com/freeCodeCamp/freeCodeCamp/issues/38299

Right now we want to be able to bring the Chinese translation for the Responsive Web Design certification at par with the English version.

All PRs will go thru the dev team for a little bit while the Chinese translation team if working on fixing the issue. This limitation will be lifted soon. 

Thanks for your patience.

**Comment by raisedadead** [comment]
**Code Freeze for Coding Interview Prep Challenges**

There is a path chance incoming on the Python branch, which brings in a freeze on changes to CIP challenges. This is temporary and once we merge the python branch to master you will be unblocked.

`master`

![image](https://user-images.githubusercontent.com/1884376/82824632-49ad5c00-9ec7-11ea-9e6a-4c7e978de030.png)


`next-python-projects`

![image](https://user-images.githubusercontent.com/1884376/82824608-41552100-9ec7-11ea-9610-5e5791ccec5c.png)

**Comment by raisedadead** [comment]
Code Freeze for Information Security and Quality Assurance Challenges

We are splitting up the above cert into two different certifications with additional updates to them. (**You will still keep your past certificate**).

Hence this brings a freeze on the InfoSec and QA challenges.

**Comment by raisedadead** [comment]
Code freeze has been removed on all PRs pending the migration of the schema. We want to land as many PRs on `master` as we can before running a script to update all challenge files.

/cc @freeCodeCamp/dev-team

**Comment by raisedadead** [comment]
*Code Freeze on all PRs (both curriculum and platform) due to incoming new parser for the curriculum. This is also going to change the challenge schema and all curriculum files.*

We will update everyone as soon as this freeze is lifted.

**Comment by raisedadead** [comment]
**Code Freeze for TyprScript Migration of the Client**

Currently, we are working on landing #42624 onto the `main` development branch. To enable us to QA this closely we are enforcing a freeze on both `next` & `main` branches of this repository.

The @freeCodeCamp/dev-team is working on resolving & pushing fixes for outstanding issues that are being discussed in #42660

**What this means:**

- While this is activity is in progress we are not allowing any new feature PRs to be merged into both branches. We are also not allowing any bug fix PRs to be merged to the main branch to help avoid the tedious rebasing cycles.

- If you are interested in helping fix any stability related bugs to the `next` branch, we are welcoming you to make a PR targeting the same.

- Finally, if you are working on anything other than the issue listed in the bug thread mentioned earlier, please continue to do so. We will take care of reviewing them once we are happy with the state of the `next` branch. We appreciate your understanding of the delays.

Thanks.

**Comment by ieahleen** [comment]
#42596 has been merged, is the Code Freeze over?

**Comment by ShaunSHamilton** [comment]
@ieahleen Largely, yes, the ❄️ is over. However, the changes are not on production yet. So, have been easing out of the Code Freeze.

**Comment by ahmadabdolsaheb** [comment]
Do we still need this?

**Comment by ojeytonwilliams** [comment]
We've just been pinning when there's a freeze and unpinning when it's done. I think we should keep the issue around for next time.
